docs/sccm/preparation/issue-323-client-updates-corpus.md

```
@@ -0,0 +1,205 @@
+Current preparation replay:
+
+```bash
+cargo test --locked -p cmtraceopen-parser --test sccm_client_updates_fixture_contract
+cargo test --locked -p cmtraceopen-parser
+cargo clippy --locked -p cmtraceopen-parser --all-targets -- -D warnings
+cargo check --locked -p cmtraceopen-parser --target wasm32-unknown-unknown
+npx tsc --noEmit
+```
+
+Native Windows source discovery/capture is not exercised by this slice. Issue
+#323 must remain open for production implementation, shared-interface review,
```

markdownlint-cli2 (0.23.1)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)